Libraries/Text/TextInput/Singleline/RCTUITextField.m

```diff
@@ -321,15 +315,8 @@ - (void)setSecureTextEntry:(BOOL)secureTextEntry
 {
   NSMutableDictionary<NSAttributedStringKey, id> *textAttributes = [_defaultTextAttributes mutableCopy] ?: [NSMutableDictionary new];
 
-  if (self.placeholderColor) {
```

Comment:
Why are we deviating from core here?   For iOS, shouldn't we only set the value if self.placeholderColor is non-nil and remove the key otherwise?

Reply:
Hmm, I don’t recall if I had found an issue on iOS that this fixes—I don’t believe so—but it was definitely just a simplification of the code, as the end result is the same. The goal being, of course, to move this upstream so we won't need any more ifdefs here.

Reply:
I think its fine.  I can't imagine a customer (particular Office code) what would leave the placeholder color default and expect it to be black -- its not system theme friendly.